app/pages/cif/project-revision/[projectRevision]/project-revision-view.tsx

```diff
@@ -0,0 +1,168 @@
+    session {
+      ...DefaultLayout_session
+    }
+    projectRevision(id: $projectRevision) {
+      id
+      revisionType
+      createdAt
+      cifUserByCreatedBy {
+        fullName
+      }
+      updatedAt
+      cifUserByUpdatedBy {
+        fullName
+      }
+      typeRowNumber
+      # eslint-disable-next-line relay/unused-fields
+      comments
+      projectByProjectId {
+        latestCommittedProjectRevision {
+          ...TaskList_projectRevision
+        }
+      }
+    }
+    allRevisionTypes {
+      edges {
+        node {
+          type
+        }
+      }
+    }
+  }
+`;
+export function ProjectRevisionView({
+  preloadedQuery,
+}: RelayProps<{}, projectRevisionViewQuery>) {
+  const { session, projectRevision, allRevisionTypes } = usePreloadedQuery(
+    ProjectRevisionViewQuery,
+    preloadedQuery
+  );
+  const taskList = (
+    <TaskList
+      projectRevision={
+        projectRevision.projectByProjectId.latestCommittedProjectRevision
+      }
+      mode={"view"}
+      projectRevisionUnderReview={projectRevision}
+    />
+  );
+  return (
+    <>
+      <DefaultLayout session={session} leftSideNav={taskList}>
+        <header>
+          <h2>
+            {projectRevision.revisionType} {projectRevision.typeRowNumber}
+          </h2>
+        </header>
+        <div>
+          <FormBase
+            id={`form-${projectRevision.id}`}
+            tagName={"dl"}
+            className="project-revision-view-form"
+            schema={
+              createProjectRevisionViewSchema(allRevisionTypes) as JSONSchema7
+            }
+            uiSchema={createProjectRevisionUISchema()}
+            ObjectFieldTemplate={EmptyObjectFieldTemplate}
+            theme={readOnlyTheme}
+            formData={projectRevision}
+          ></FormBase>
+          <div className="revision-record-history-section">
+            <dt>Revision record history</dt>
+            {projectRevision.updatedAt && (
+              <dd>
+                <em>Updated by </em>
+                {projectRevision.cifUserByUpdatedBy?.fullName}
+                <em>on </em>
+                {getLocaleFormattedDate(
+                  projectRevision.updatedAt,
+                  "DATETIME_MED"
+                )}
+              </dd>
+            )}
+            <dd>
+              <em>Created by </em>
+              {projectRevision.cifUserByCreatedBy?.fullName}
+              <em>on </em>
+              {getLocaleFormattedDate(
+                projectRevision.createdAt,
+                "DATETIME_MED"
+              )}
+            </dd>
+          </div>
+        </div>
+      </DefaultLayout>
+      <style jsx>{`
+        div :global(.definition-container) {
+          display: flex;
+          flex-direction: column;
+          gap: 0.5rem;
+        }
+        div :global(.radio > label) {
+          font-weight: normal;
+        }
+        div :global(input[type="radio"]) {
+          margin-right: 1em;
+        }
```

Comment:
@pbastia we might not be able to use the first selector in `ReadOnlyFieldTemplate` since we are always using `flex-row` to display read-only fields. but the rest is definitely a common style.